dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -966,11 +1060,7 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 											</ArticleContainer>
 										</div>
 									) : (
-										<Accordion
-											supportsDarkMode={false}
-											accordionTitle="Live feed"
-											context="liveFeed"
-										>
+										<div css={paddingBody}>
```

Comment:
Is the removal of the accordion something we could break out?

Reply:
yep - i'll revert this